src/utils/findProductById.js

```diff
@@ -0,0 +1,11 @@
+/*
+Receives an array where we'll loop through to find the element
+that matches the id
+
+The array MUST contain an id property!
+*/
+
+export const findProductById = (array, itemId) => {
+  const product = array?.find((item) => item.id === itemId);
```

Comment:
Nice use of the optional chaining operator! 

Just one thing to consider: `array` probably can be renamed as `productList` or `collection` or something more meaningful.

Reply:
I figured this function could have its name changed, as giving it the "product" might reduce its usage when all it does is find an item within an array an return it